src/extensions/default/SVGCodeHints/SVGAttributes.json

```diff
@@ -11,6 +11,15 @@
     "animate/fill": {
         "attribOptions": ["freeze", "remove"]
     },
+    "animateColor/fill": {
+        "attribOptions": ["freeze", "remove"]
+    },
```

Comment:
OK, that makes sense. Do you want to add color name support in this PR so that it's clear why it was done this way?